Pod/Classes/Acknowledgements/AcknowlegementsListViewModel.swift

```diff
@@ -71,7 +71,10 @@ private extension AcknowledgementsListViewModel {
             return []
         }
         // First and last elements in the settings bundle are not needed (title and empty row).
-        let acknowledgements = specifiers.suffix(from: 1).dropLast().flatMap(AcknowledgementViewModel.init(dictionary:))
+        let acknowledgements = specifiers.suffix(from: 1).dropLast().flatMap(AcknowledgementViewModel.init(dictionary:)).sorted {
```

Comment:
Since we're touching this line anyway, I think there should be able to grab all but the first and last items in a way that's O(n) instead of O(2n). I think it would be along the lines of `specifiers[1..<specifiers.count]`. There might be bounds checking needed, but I don't think so, because we don't care about cases where it's empty (since you aren't using this if you have no acknowledgements).